scripts/datageneration/tests/test_retrieve_combination.py

```diff
@@ -9,13 +9,12 @@
 
 class TestCombinationRetriever(unittest.TestCase):
     def setUp(self):
-        self.retriever = CombinationRetriever(source='datageneration/tests/data/Primary_Keys_test.xlsx',
+        self.retriever = CombinationRetriever(source='datageneration/tests/data/Primary_Keys_filtered9.xlsx',
```

Comment:
@dendelyne could you please keep the name of this file "Primary_Keys_test.xlsx". Otherwise every time you need to change this test script and have to commit. But instead you should update Primary_Keys_test.xlsx with the last version.